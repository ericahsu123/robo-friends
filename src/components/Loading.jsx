import React from 'react';

const Loading = () => {
  return (
    <div>
      <div className="ui active inverted dimmer">
        <div className="ui text loader">Loading</div>
      </div>
    </div>
  );
};

export default Loading;
