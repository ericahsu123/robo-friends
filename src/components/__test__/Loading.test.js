import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import Loading from '../Loading';

test('<Loading/> matches snapshot', () => {
  const tree = renderer.create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});
