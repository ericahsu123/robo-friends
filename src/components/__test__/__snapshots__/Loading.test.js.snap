// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading/> matches snapshot 1`] = `
<div>
  <div
    className="ui active inverted dimmer"
  >
    <div
      className="ui text loader"
    >
      Loading
    </div>
  </div>
</div>
`;
